config/main.py

```diff
@@ -6,6 +6,8 @@
 import json
 import subprocess
 import netaddr
+import logging
+import logging.handlers
```

Comment:
These imports don't appear to be used. Remove?

Reply:
@jleveque : Done. All changes are completed.